variables.tf

```diff
@@ -42,14 +53,7 @@ variable "instance_type" {
 
 variable "key_name" {
   description = "SSH key name in your AWS account for AWS instances."
-    default   = "amazonhosts"
-}
-
-variable "key_path" {
-  description  = "Path to the private key specified by key_name."
-    default    = {
-      key_path = "/Users/alaric/amazonhosts.pem"
-    }
+  default     = "amazonhosts"
```

Comment:
works for me! `amazonhosts` probably should never have ended up in the module in the first place 😊